src/formatter.js

```diff
@@ -52,21 +52,29 @@ module.exports = function(results) {
     output += '##teamcity[testStarted name=\'' + reportName + ': ' +
                escapeTeamCityString(result.filePath) + '\']\n';
 
-    var messageList = [];
+    var errorsList = [];
+    var warningsList = [];
+    var userMessage;
 
     messages.forEach(function(message) {
-      if (message.severity === 2) {
-        messageList.push(
-          'line ' + message.line +
-          ', col ' + message.column + ', ' + message.message
-        );
+      userMessage = 'line ' + (message.line || 0) +
+          ', col ' + (message.column || 0) + ', ' + message.message + (message.ruleId ||  message.ruleId ? ' (' + message.ruleId + ')' : '');
```

Comment:
The truthy part of the ternary will never never be hit since if ruleId exists, it won't get to the ternary operation in the first place

what about - without the `message.ruleId ||` part:

`(message.ruleId ? ' (' + message.ruleId + ')' : '')`

?